pipelines/build/openjdk_pipeline.groovy

```diff
@@ -72,26 +72,23 @@ node ("master") {
         checkout scm
     }
 
-    // Get JDK head number so we can identify which config to pull in for JDK head
-    String headVersion = ""
-    if (javaToBuild == "jdk") {
-        def JobHelper = library(identifier: 'openjdk-jenkins-helper@master').JobHelper
-        println "Querying Adopt Api for the JDK-Head number (tip_version)..."
-
-        def response = JobHelper.getAvailableReleases(this)
-        headVersion = response.getAt("tip_version")
-        println "Found Java Version Number: ${headVersion}"
-    }
-
     // Load buildConfigFilePath. This is where jdkxx_pipeline_config.groovy is located. It contains the build configurations for each platform, architecture and variant.
-    def buildConfigFilePath = (params.buildConfigFilePath) ?: "${DEFAULTS_JSON['configDirectories']['build']}/${javaToBuild}${headVersion}_pipeline_config.groovy"
+    def buildConfigFilePath = (params.buildConfigFilePath) ?: "${DEFAULTS_JSON['configDirectories']['build']}/${javaToBuild}_pipeline_config.groovy"
```

Comment:
This change will cause the warning every time we try to build an update project (e.g. jdk8u).

Since the separate pipeline files didn't have this behaviour, can I suggest adding a check for both the u and non-u files before we go declaring a warning?

Reply:
updated PTAL

Reply:
Thanks works. Thanks. :)